gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -92,7 +92,7 @@ pub struct Htlc {
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Onion {
     #[serde(default)]
-    pub short_channel_id: Option<u64>,
+    pub short_channel_id: Option<String>,
```

Comment:
`cln-plugin` returns the scid in human readable format. I initially defined the spec here to expect a u64 which is what is defined in bolt spec

Reply:
Using `u64` was just incorrect. Core Lightning has never returned a `u64` for this field. We really need to upstream these structs to avoid errors like this in the future: https://github.com/fedimint/fedimint/issues/1245.